assets/js/plan_template.js

```diff
@@ -149,8 +150,9 @@ async function parseResponse(response) {
     console.log("Raw JSON response is", response);
 
     $(function () {
-        if (response["travel_plans"].length > 0) {
-            document.getElementById("results").classList.remove("d-none");
+        const resultsTable = document.getElementById("results");
+        if (response["travel_plans"]?.length > 0) {
```

Comment:
Curious what does "?" do here in if ().

Reply:
Prevent the variable before `?` equals `null`

Reply:
Cool, learned something new! I have also searched a bit, and it is called optional chaining in JS guide. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining